src/service/recorder/influxDataRecorder.ts

```diff
@@ -1,41 +0,0 @@
-import { InfluxDB } from 'influx';
```

Comment:
Why did you rename the file? The idea was that this recorder could be used to store any characteristic to Influx. Characteristic specific data transformation is supposed to be done by an appropriate Transformer function.

Reply:
Because this recorder type contains x,y,z fields and is therefore gravity specific, right?

Reply:
Yes because I was to lazy to implement it properly ;) Will fix